scripts/common.sh

```diff
@@ -185,7 +181,21 @@ function diff_values
     shift
     shift
 
-    diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    if ! diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    then
+        if [ "${DIFFVIEW:-}" == "" ]
+        then
+            printError "ERROR: values differ: '${value1}' vs. '${value2}'"
+            printError "Expected:"
+            printError "${value1}"
+            printError "Obtained:"
+            printError "${value2}"
+        else
+            # Use user supplied diff view binary
+            "$DIFFVIEW" "${file1}" "${file2}"
```

Comment:
In this case I'd still print the error, just without showing full diff on the console.